docs/docs/general/02-Getting Started/getting-started.md

```diff
@@ -36,11 +36,4 @@ Quick explanation of all the dependencies:
 
 If you don't like Jest, feel free to use another test framework, e.g., [Jasmine](https://jasmine.github.io/) or [Mocha](https://mochajs.org/).
 
-
-## Configure
-
-Currently, you can use our lib without any configuration. But in the future, as we need to prevent misuse of our api, we may need you to create some credentials through our [askui user portal](https://app.askui.com/) (still, totally free) and [configure our library to use these credentials](../../api/06-Configuration/askui-ui-control-client.md#credentials) for authenticating and authorising with our api.
-
-Please see [our documention on how to signup and create the credentials](../08-askui%20User%20Portal/signup.md).
-
```

Comment:
I would keep it but more like a _PS_ section (later) so that people don't have to read it but it is there so some may already create their credentials.

Reply:
Sounds reasonable. I would like to call it "API Access Token" or do you prefer "Configure" here?

Reply:
Both work from my perspective :)

Reply:
Ok, then I decide to leave it as committed 😺